libs/functions/calculate_projections.py

```diff
@@ -16,6 +16,10 @@
 from libs.enums import Intervention
 from libs.constants import NULL_VALUE
 
+from pandarallel import pandarallel
+pandarallel.initialize(progress_bar=True)
```

Comment:
I would prefer not to do this initialization at the module level.  would it be possible to add this in to probably `run.py`? We don't yet have a common place to initialize all of these things - may be worth creating a function that does that initialization (would setup sentry, logging and now this)

Reply:
Makes sense.  Should we do that in a separate module level init branch?

Reply:
can we prefer to throw this in `run.py` for now